samples/bluetooth/unicast_audio_server/src/main.c

```diff
@@ -205,15 +252,54 @@ static void stream_disconnected(struct bt_audio_stream *stream, uint8_t reason)
 	printk("Audio Stream %p disconnected (reason 0x%02x)\n", stream, reason);
 }
 
+
+#if defined(CONFIG_LIBLC3CODEC)
+
+static void stream_recv_lc3_codec(struct bt_audio_stream *stream, struct net_buf *buf)
+{
+	uint8_t bit_error_detect = 0;
+	uint8_t err;
+	/* TODO: If there is a way to know if the controller supports indicating errors in the
+	 *       payload one could feed that into bad-frame-indicator. The HCI layer allows to
+	 *       include this information, but currently there is no controller support.
+	 *       Here it is assumed that reveiving a zero-length payload means a lost frame -
+	 *       but actually it could just as well indicate a pause in the stream.
+	 */
+	uint8_t bad_frame_indicator = buf->len == 0 ? 1 : 0;
```

Comment:
```suggestion
	const uint8_t bad_frame_indicator = buf->len == 0 ? 1 : 0;
```

Reply:
Does `Lc3Decoder_run` take `bad_frame_indicator` as a `uint8_t` or `bool`?

Reply:
uint8_t for some reason